open-x-htb.js

```diff
@@ -312,7 +312,8 @@ function OpenXHtb(configs) {
             var sizeKey = Size.arrayToString(curReturnParcel.size);
 
             if (bidDealId !== '') {
-                curReturnParcel.targeting[__baseClass._configs.targetingKeys.pm] = [sizeKey + '_' + bidDealId];
+                curReturnParcel.targeting[__baseClass._configs.targetingKeys.pm] = [sizeKey + '_' + bidPrice];
+                curReturnParcel.targeting[__baseClass._configs.targetingKeys.pmid] = [sizeKey + '_' + bidDealId];
```

Comment:
According to your official documentation: https://github.com/indexexchange/open-x-adapter/blame/master-v2/README.md#L235-L236, pm should be the bid cpm while pmid should be the bid deal id. The code here seems not correct. Please confirm this change wouldn't cause problems for current lineitem triggering.(aka, no client is using pm=<dealid> as lineitem key-value targeting).

Reply:
The code in the documentation above is correct, however it is for a different template than OpenX is currently using (functionally, the templates are all the same). The changes you propose would cause the existing line items to stop triggering. We suggest that the changes be undone including the ones made to the `__profile.targetingKeys` object below.

Reply:
Thanks for the clarification. I've restored the changes and added a comment in README.md.